### ISSUE TITLE
Warnings occur during build

### ISSUE THREAD

**Issue by projectX21** [ghiscoding/Angular-Slickgrid#526]
### Your Environment

| Software          | Version(s) |
| ----------------- | ---------- |
| Angular           | 10.0.3        | 
| Angular-Slickgrid | 2.19.0        | 
| TypeScript        | 3.9.6       | 
| Operating System  | Debian Stretch | 
| NPM/Node/Yarn     | x.y        | 

### Context or Topic 

During the build I get some warnings with Angular 10.0.3:

Entry point 'angular-slickgrid' contains deep imports into '/node_modules/slickgrid/plugins/slick.cellrangedecorator', '/node_modules/slickgrid/plugins/slick.cellrangeselector', '/node_modules/slickgrid/plugins/slick.cellselectionmodel', '/node_modules/jquery-ui-dist/jquery-ui', '/node_modules/slickgrid/lib/jquery.event.drag-2.3.0', '/node_modules/slickgrid/lib/jquery.mousewheel', '/node_modules/slickgrid/slick.core', '/node_modules/slickgrid/slick.grid', '/node_modules/slickgrid/slick.dataview', '/node_modules/flatpickr/dist/types/locale'. This is probably not a problem, but may cause the compilation of entry points to be out of order.

And:

WARNING in /src/app/app.module.ts depends on 'angular-slickgrid'. CommonJS or AMD dependencies can cause optimization bailouts.
For more info see: https://angular.io/guide/build#configuring-commonjs-dependencies

WARNING in /node_modules/angular-slickgrid/__ivy_ngcc__/fesm2015/angular-slickgrid.js depends on 'jquery-ui-dist/jquery-ui'. CommonJS or AMD dependencies can cause optimization bailouts.
For more info see: https://angular.io/guide/build#configuring-commonjs-dependencies

WARNING in /node_modules/angular-slickgrid/__ivy_ngcc__/fesm2015/angular-slickgrid.js depends on 'excel-builder-webpacker'. CommonJS or AMD dependencies can cause optimization bailouts.
For more info see: https://angular.io/guide/build#configuring-commonjs-dependencies

I don't have the knowledge here, but is it something angular-slickgrid has to fix, or do I have to search a project based solution?

Do those messages occur with Angular 9, too?

Thank you in advice!

**Comment by ghiscoding** [comment]
Yes these are normals, I dynamically load all the SlickGrid extensions [here](https://github.com/ghiscoding/Angular-Slickgrid/blob/master/src/app/modules/angular-slickgrid/extensions/extensionUtility.ts#L29), depending on which flags you use in the grid options, with `require('slickgrid/plugins/slick....');` and Angular complains about that. It was also mentioned in this [issue](https://github.com/ghiscoding/Angular-Slickgrid/issues/413)

**Comment by ghiscoding** [comment]
@projectX21 
Took a quick look to see if there was a way to get rid of these warnings, which as I said earlier I'm fine with them... and I found this Angular [issue](https://github.com/angular/angular/issues/35615) which mention a way to bypass the warnings. 

I added a section in the Readme [here](https://github.com/ghiscoding/Angular-Slickgrid#build-warnings-angular-8) to show how to get rid of these warnings. I also updated my own Angular-Slickgrid-Demos in this [commit](https://github.com/ghiscoding/angular-slickgrid-demos/commit/1fe8092bcd2e99ede5ab048f4a7ebe6254e4bee0) and finally I got no more warnings while doing a Prod build (which I do every time I push a new version).

Cheers
